en/docs/deploy/elk-analytics-installation-guide.md

```diff
@@ -398,28 +398,39 @@ Once the above steps are completed, role mapping needs to be configured in Kiban
 
 #### Configure role mapping in Kibana
  
-1. Login to Kibana using basic authentication.
+1. Edit the following configuration in `kibana.yml` file.
```

Comment:
```suggestion
1. In your Kibana installation, navigate to the **config** directory and open **kibana.yml** file
2. Add the following configuration to the **kibana.yml** file.
```

Reply:
Addressed in [8f3145a](https://github.com/wso2/docs-is/pull/3092/commits/8f3145a3f5a76ae3048fd3430e61429efb526df5)